test_syndiffix/oneModel.py

```diff
@@ -149,7 +149,23 @@ def makeMetadata(df):
 def getTopFeatures(featuresJob, numFeatures):
     return featuresJob['features'][:numFeatures]
 
-def getFeaturesByThreshold(featuresJob, featureThreshold):
+def getMlFeaturesByThreshold(featuresJob, featureThreshold):
+    # We always include the top feature
+    features = [featuresJob['features'][0]]
+    topScore = featuresJob['cumulativeScore'][0]
+    if topScore == 0:
+        # Don't expect this, but you never know
+        return features
```

Comment:
I do a topScore of zero when there is only one other feature and running ML doesn't make sense for such datasets.

Reply:
ok, will keep in mind